scripts/linters/general_purpose_linter.py

```diff
@@ -571,17 +584,23 @@ def check_file_type_specific_bad_pattern(filepath, content):
      Returns:
         failed: bool. True if there is bad pattern else false.
         total_error_count: int. The number of errors.
+        summary_messages: list(str). Summary of failed messages.
```

Comment:
Please change this to follow the general docstring pattern

Reply:
Done.